src/icrar/leap-accelerate/algorithm/cuda/PhaseRotate.cu

```diff
@@ -294,15 +295,15 @@ namespace cuda
         );
```

Comment:
Just above these lines there are a couple of `assert`s that fail to compile with the latest changes, they need to be changed to use `GetAvgData` and `GetVis`